external/mmdetection/detection_tasks/apis/detection/ote_utils.py

```diff
@@ -114,7 +114,7 @@ def on_epoch_end(self, epoch, logs=None):
         if score is not None:
             self.update_progress_callback(self.get_progress(), score=float(score))
         else:
-            self.update_progress_callback(int(self.get_progress()))
+            self.update_progress_callback(self.get_progress())
```

Comment:
No need to have these two branches anymore if you stick to use only training update progress callback.